CHANGELOG.md

```diff
@@ -22,6 +22,14 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 - Crash when using unrecognized remote control (#1685)
 - Fixed mismatch between internal and display state after clearing data (#1691)
 
+## [3.3] - 2019-01-31
+*Released to all devices except Fire TV 4K.*  
+  
+Version-bump only - no change from v3.2.5
+
+## [3.2.5] - 2019-01-16
```

Comment:
Even though 3.2.3 & .4 were not released, I think we should include them here with a brief summary that they were not released and why not: it prevents people from wondering, "What happened to these builds?"